odm_sdk/settings/genestack_config.py

```diff
@@ -5,9 +5,9 @@
 
 import sys
 
-from genestack_client import GenestackException
-from genestack_client.settings.genestack_user import User
-from genestack_client.utils import ask_confirmation
+from odm_sdk import GenestackException
+from odm_sdk.settings.genestack_user import User
```

Comment:
since we are renaming genestack to odm in the library name, possibly we need to do the same in the code? For example genestack_user -> odm_user

Reply:
It will lead us to rename [this](https://github.com/genestack/python-client/blob/946e58e8593d3908e1b49cf0c121af72e18299aa/odm_sdk/settings/genestack_config.py#L12-L16) piece of code. And a lot more places. 

I just tried to make the same prefix to have a comfortable work in bash using [Tab] and that's it.

For me, it's better to revert changes in CLI rather than continue with rewriting the whole sdk.

Reply:
>revert changes in CLI
do you mean this PR or which changes?

Reply:
I meant [those](https://github.com/genestack/python-client/blob/54d83016dfa645da8c22297a3d72e20cb486a237/setup.py#L30-L31) lines.